docs/element-definitions.md

```diff
@@ -607,25 +656,19 @@ Type Element matching only lists of one of `string`, `number`.
 ---
 
 ### Member Element
-Type with domain of all _properties_.
-
-Quoting from the [Element section](#element):
-> A _property_ is a tuple (`key`, `value`) where
-> - `key` SHALL be a non-empty, finite character string
-> - `value` SHALL be an _Element_
-> - two properties SHALL be equal if their keys are.
+Type with domain of all [_properties_](#property).
 
 #### Template
 - `element` - `"member"`
 - `attributes`
   - `typeAttributes` ([Array](#array-element)[[String](#string-element)])
-    - `required` ([String](#string-element)) - Property MUST be present in Object containing it. I.e. restricts the domain of the containing Object Element type to one containing this property.
-    - `optional` ([String](#string-element)) - Property MAY NOT be present in Object containing it. I.e. expands the domain of the containing Object Element type to one not containing this property.</p>
+    - `required` ([String](#string-element)) - Property MUST be present in value represented by the containing [Object Element](#object-element). I.e. restricts the domain of the containing Object Element type to one containing this property.
+    - `optional` ([String](#string-element)) - Property MAY NOT be present in value represented by the containing [Object Element](#object-element). I.e. expands the domain of the containing Object Element type to one not containing this property.
   - `variable` - ([Boolean](#boolean-element)) - Property key SHALL be interpreted as a variable name instead of a literal name
   - `validation` - _reserved for future use_
 - `content`
-  - `key` - an Element representing a key; SHOULD be a [String Element](#string-element)
-  - `value` - an Element representing the value
+  - `key` - An Element representing a key; SHOULD be a [String Element](#string-element)
```

Comment:
I think it would be worth mentioning here or below somewhere that a `key` is expected to be there. A member element MUST contain a key, although it MAY contain a value, you can have members with keys without values. You cannot have a member without a key.